Documentation/HowToGuides/AddingACollisionIgnorer/README.md

```diff
@@ -0,0 +1,76 @@
+# Adding A Collision Ignorer
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+The `Input Mutator CoThe Collision Ignorer prefabs provides a way of allowing two or more objects to ignore collisions with each other.lliisonIgnorer` provides a way of allowing two or more objects to not interact with each other.
+
+## Prerequisites
+
+* [Install the Tilia.Mutators.CollisionIgnorer.Unity] package dependency in to your Unity project.
+
+## Let's Start
+
+### Step 1
+
+Create a new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 0`, `Z = 0`
+
+![Adding A Cube](assets/images/AddingACube.png)
+
+### Step 2
+
+Create another new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 2`, `Z = 0`
+
+![Adding The Second Cube](assets/images/AddingTheSecondCube.png)
+
+### Step 3
+
+Click the `Add Component` button and select the Rigidbody component to add a Rigidbody to Cube (1). 
+
+![Adding A Rigidbody](assets/images/AddingARigidbody.png)
+
+Play the Unity scene and you should see `Cube(1)` land on top of `Cube` because the colliders on each cube will prevent them passing through each other.
+
+![Playing The Unity Scene With The Rigidbody](assets/images/PlayingTheUnitySceneWithTheRigidbody.png)
+
+### Step 4
+
+Expand the `Tilia Mutators CollisionIgnorer` Package directory in the Unity Project window and select then `Packages -> Tilia Mutators CollisionIgnorer Unity -> Runtime -> Prefabs` directory then drag and drop the `Mutators.ColliisonIgnorer` prefab into the Hierarchy window.
+
+![Drag And Drop Collision Ignorer Into Hierachy](assets/images/DragAndDropCollisionIgnorerIntoHierachy.png)
+
+
+### Step 5
+
+Select the Mutators.CollisionIgnorer GameObject in the Unity Hierarchy window then increase the Sources -> Elements 
+-> Size property by 1 on the Collision Ignorer component.
+Drag and drop the Cube GameObject from the Unity Hierarchy window into the newly displayed Element 0 field within the Sources -> Elements parameter on the Collision Ignorer component.
+
+![Drag And Drop Cube Into Element](assets/images/DragAndDropCubeIntoElement.png)
+
+### Step 6
+
+Increase the `Targets -> Elements -> Size property` by `1` on the `Collision Ignorer` component.
+
+The size property will be `0` by default so change it to `1`.
+
+Drag and drop the `Cube (1)` GameObject from the `Unity Hierarchy` window into the newly displayed Element `0` field within the `Targets -> Elements` parameter on the `Collision Ignorer` component.
+
+![Drag And Drop Second Cube Into Element](assets/images/DragAndDropSecondCubeIntoElement.png)
+
+Play the Unity scene you should see `Cube(1)` fall right through `Cube`.
```

Comment:
Update this text to read:

Play the Unity scene you should see `Cube (1)` fall right through `Cube` this is because the collisions between the two GameObjects is now being ignored.